src/xlora/xlora_insertion.py

```diff
@@ -393,21 +393,21 @@ def save_pretrained(
 
         conf = classifier.config.__dict__.copy()
         del conf["device"]
+
```

Comment:
All adapters are always saved. Previous models may only be loaded via specifying override adapter list

Reply:
Old models may still be used. This is OK